server/data/mappers/createOrUpdateInductionDtoMapper.test.ts

```diff
@@ -11,21 +11,23 @@ import PersonalInterestsValue from '../../enums/personalInterestsValue'
 import SkillsValue from '../../enums/skillsValue'
 import TypeOfWorkExperienceValue from '../../enums/typeOfWorkExperienceValue'
 import WorkInterestsValue from '../../enums/workInterestsValue'
-import {
-  aCreateLongQuestionSetInductionDto,
-  aCreateShortQuestionSetInductionDto,
-} from '../../testsupport/createInductionDtoTestDataBuilder'
 import toCreateOrUpdateInductionDto from './createOrUpdateInductionDtoMapper'
+import {
+  anUpdateLongQuestionSetInductionDto,
+  anUpdateShortQuestionSetInductionDto,
```

Comment:
Using the update builder methods, rather than the create ones, means that the mapping of the `reference` fields is also tested.